src/pykeen/ablation/__init__.py

```diff
@@ -1,10 +1,31 @@
 # -*- coding: utf-8 -*-
 
-"""Ablation studies in PyKEEN."""
+"""Ablation studies in PyKEEN.
 
-from .ablation import prepare_ablation, prepare_ablation_from_config
+  "If :func:`pykeen.pipeline.pipeline` were making a maze, :func:`pykeen.hpo.hpo_pipeline` were getting
+  killed by your dead wife, then :func:`pykeen.ablation.ablation_pipeline` is washing up on
+  a beach." -Christopher Nolan
+
+TODO for Mehdi!
```

Comment:
@mali-git TODO

Reply:
Done in 7e1bafa0ca149b941c001c80fc24f0048a4c8682-8ad9dfe31c7f590b6b93e865c18e7061f6935816.